app/graphql/types/rock_and_pebble_type.rb

```diff
@@ -1,20 +1,20 @@
 module Types
-    class RockAndPebbleType < Types::BaseObject
-        field :id, ID, null: false
-        field :rock, Types::UserType, null: false
-        field :pebble, Types::UserType, null: false
-        field :active, Boolean, null: false
-        field :my_rocks, [Types::UserType], null: true
-        field :my_pebbles, [Types::UserType], null: true
+  class RockAndPebbleType < Types::BaseObject
+      field :id, ID, null: false
+      field :rock, Types::UserType, null: true
+      field :pebble, Types::UserType, null: true
+      field :active, Boolean, null: false
+      field :my_rocks, [Types::UserType], null: true
+      field :my_pebbles, [Types::UserType], null: true
 
-        def my_rocks
-          user = User.find(object.pebble_id)
-          user.rocks
-        end
+  def my_rocks
```

Comment:
@linda-le1 you were asking if these should be deleted? I’m technically using them on one of my queries but I’m wondering if my query can be refactored to just use a user type as well so we could dry this up a bit. It will be good to talk about this tomorrow morning!